package.json

```diff
@@ -81,7 +82,7 @@
     "crypto-randomuuid": "^1.0.0",
     "dc-polyfill": "^0.1.4",
     "ignore": "^5.2.4",
-    "import-in-the-middle": "^1.7.4",
+    "import-in-the-middle": "https://github.com/DataDog/import-in-the-middle#dbaca128f6eb517d71bcf7e6b6b45f21a07128a4",
```

Comment:
using this commit: https://github.com/DataDog/import-in-the-middle/commit/dbaca128f6eb517d71bcf7e6b6b45f21a07128a4 because I haven't been able to make IITM work with that line